.github/workflows/reviewdog.yml

```
@@ -0,0 +1,29 @@
+name: reviewdog
+on:
+ pull_request:
+     branches:
+      - master
+      - adding-review-dog
+
+jobs:
+  reviewdog:
+    name: reviewdog
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
```

```suggestion
        uses: actions/setup-go@v3
```
https://github.com/marketplace/actions/setup-go-environment#v3